model/job-management/src/main/proto/beam_job_api.proto

```diff
@@ -123,6 +126,22 @@ message CancelJobResponse {
   JobState.Enum state = 1; // (required)
 }
 
+// A subset of info provided by ProvisionApi.ProvisionInfo
+message JobInfo {
+  string job_id = 1; // (required)
+  string job_name = 2; // (required)
+  google.protobuf.Struct pipeline_options = 3; // (required)
+  JobState.Enum state = 4; // (required)
```

Comment:
I debated whether or not to include the state here.  

Reason for including it:  It's cheap to get, and it's the kind of info that any web client would want to provide when displaying an overview of jobs (see flink web ui).  Providing it obviates the need for these clients to issue a followup query to get the state of each job.

Reason for excluding it: It's the only value here that is not an intrinsic, immutable property of a job.  i.e. it changes over time, and so conceptually, including it means that the JobInfo becomes invalid over time.  Removing it would make it a close fit for `beam.runners.fnexecution.provisioning.JobInfo`.

Reply:
I think we can keep it for now.